app/cinema/hall.py

```diff
@@ -0,0 +1,18 @@
+from app.people.cinema_staff import Cleaner
+
+
+class CinemaHall:
+    def __init__(self, number: int) -> None:
+        self.number = number
+
+    def movie_session(self, movie_name: str,
+                      customers: list,
+                      cleaning_staff: Cleaner) -> None:
```

Comment:
Start every argument from a new line
```suggestion
    def movie_session(self,
                      movie_name: str,
                      customers: list[Customer],
                      cleaning_staff: Cleaner) -> None:
```